crates/backend/src/codegen.rs

```diff
@@ -1334,9 +1348,15 @@ impl<'a, T: ToTokens> ToTokens for Descriptor<'a, T> {
             return;
         }
 
-        let name = Ident::new(&format!("__wbindgen_describe_{}", ident), ident.span());
+        let name = Ident::new(
+            &format!("__WBINDGEN_DESCRIBE_{}", ident.to_string().to_uppercase()),
+            ident.span(),
+        );
```

Comment:
The CLI also relies on this being lowercase.

```suggestion
        let name = Ident::new(&format!("__wbindgen_describe_{}", ident), ident.span());
```